src/ngraph/check.hpp

```diff
@@ -130,8 +130,8 @@ namespace ngraph
 //
 // Take care to fully qualify the exception class name in the macro body.
 //
-// The "..." may be filled with expressions of any type that has an "operator<<" overload for\
-// insertion into std::ostream.
+// The "..." may be filled with expressions of any type that has an "operator<<"        <backslash>
+// overload for insertion into std::ostream.
 //
```

Comment:
Now I see what this is trying to say. I suggest:
`// The "..." may be filled with expressions of any type that has an "operator<<" `
`// overload that inserts backslashes into std::ostream`

Reply:
That’s not what it was trying to say. The backslash didn’t belong there in the first place, not sure how it happened. It should just be deleted.